perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -340,21 +352,46 @@ def Run(benchmark_spec):
 
   fio_command = 'sudo %s --output-format=json %s' % (fio.FIO_PATH,
                                                      REMOTE_JOB_FILE_PATH)
+
+  run_reps = FLAGS.run_for_minutes // MINUTES_PER_JOB
```

Comment:
The behavior I want is that each scenario is run for the given number of minutes, so I think this is actually the correct logic, but a confusing interface. I updated the help message for the --run_for_minutes option and added a comment here. Thanks for catching this!